remove: step.Payload removed, since it was deprecated for a long time (#115)
ovh/utask: pkg/plugins/builtin/script/README.md

## Note

> The plugin returns two objects, the `output` who is controlled by the `output_mode` configuration. It could be either the last line of your script formatted as JSON; or any given JSON in stdout/stderr inside the given starting/ending delimiters in configuration (`output_manual_delimiters`).

Reviewer: ```suggestion
The plugin returns two objects, `output` and `metadata.

`output` depends on the `output_mode` configuration. It is read from the stdout of the script, either on the last line (`output_last_line`) or between given delimiters (`output_manual_delimiters`).
```

```json
{"dumb_string":"Hello world!","random_object":{"foo":"bar"}}
```

> The `metadata` to fetch informations about plugin execution:

Reviewer: ```suggestion
`metadata` contains information about the script execution:
```
